alphapose/utils/vis.py

```diff
@@ -31,89 +59,8 @@ def get_color_fast(idx):
 
     return color
 
+def vis_frame_fast(frame, im_res, opt, format='coco'):
 
-def vis_frame_dense(frame, im_res, add_bbox=False, format='coco'):
```

Comment:
I think we can add it back, change the add_bbox to opt. Just ignore the face here.